student.rb

```diff
@@ -7,7 +7,7 @@ class Student < Person
   def initialize(age, _classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
   end
-
+  h
```

Comment:
- [ ] Please remove this unnecessary `h` from here, Users will not be able to start the app because of it.